testsuite/cluster-test/src/experiments/versioning_test.rs

```diff
@@ -101,35 +100,20 @@ impl Experiment for ValidatorVersioning {
                 150,
             )
             .await?;
-
-        info!("1. Changing the images for the instances in the first batch");
```

Comment:
Should we add this test to the land blocking suite? IMHO this test should alway pass when we have release as base image. cc @bmwill

Reply:
As discussed offline, we can merge it to land-blocking suite in a separate PR if this is intended